applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -425,8 +425,34 @@ void slm_fota_init_state(void)
 	slm_fota_info = 0;
 }
 
+#if defined(CONFIG_LWM2M_CARRIER)
+static K_SEM_DEFINE(app_fota_status, 0, 1);
```

Comment:
We have another way of performing app fota in SLM, so the name of these should indicate lwm2m.

Reply:
Added a carrier prefix